src/main.rs

```diff
@@ -1,21 +1,9 @@
 // Copyright 2020 the Deno authors. All rights reserved. MIT license.
-#![deny(warnings)]
```

Comment:
This line should stay

Reply:
this makes clippy angry for the color attributes that are not used

Reply:
I see, okay, let's remove it for now